Framework/Algorithms/src/CalculateIqt.cpp

```diff
@@ -0,0 +1,305 @@
+#include "MantidAlgorithms/CalculateIqt.h"
+#include "MantidAPI/AlgorithmManager.h"
+#include "MantidKernel/BoundedValidator.h"
+#include "MantidAPI/MatrixWorkspace.h"
+#include "MantidKernel/MersenneTwister.h"
+#include "MantidHistogramData/HistogramY.h"
+#include <cmath>
+
+using namespace Mantid::API;
+using namespace Mantid::Kernel;
+using namespace Mantid::HistogramData;
+
+namespace {
+constexpr int DEFAULT_ITERATIONS = 10;
+constexpr int DEFAULT_SEED = 23021997;
+
+MatrixWorkspace_sptr rebin(MatrixWorkspace_sptr workspace,
+                           const std::string &params) {
+  IAlgorithm_sptr rebinAlgorithm = AlgorithmManager::Instance().create("Rebin");
```

Comment:
Would `Algorithm::createChildAlgorithm("Rebin")` be better here? I believe it does the initialization and `setChild`.

Reply:
I think that would work if these functions were members, but I've kept them as free functions - they don't have an existing instance of `Algorithm` to call `createChildAlgorithm` on.